src/modules/kubernetes.rs

```diff
@@ -361,6 +370,19 @@ users: []
             })
             .collect();
 
+        let empty_dir = tempfile::tempdir()?;
+
+        let actual_env_var = ModuleRenderer::new("kubernetes")
+            .path(empty_dir.path())
+            .env("KUBECONFIG", filename.to_string_lossy().as_ref())
+            .env("TEST_K8S_ENV", "foo")
+            .config(toml::toml! {
+                [kubernetes]
+                disabled = false
+                detect_env_vars = ["TEST_K8S_ENV"]
+            })
+            .collect();
```

Comment:
Testing the negative case would probably also be nice: without the `.env("TEST_K8S_ENV", "foo")` it should be disabled.

Reply:
Have updated the tests to cover this case with the other negative tests.